docs/administration/configuration/configuring-interactive-sessions/index.md

```diff
@@ -0,0 +1,53 @@
+# Configuring interactive sessions
+
+## Interactive sessions cleanup
+
+### Automatic and scheduled closure
+
+By default, interactive sessions in REANA are kept open indefinitely. However, administrators
+have the flexibility to configure the maximum inactivity period after which interactive
+sessions will be automatically closed; the schedule of this cleanup process can be
+customized. This helps optimize resource utilization and ensure that unused sessions
+are not consuming unnecessary resources.
+
+The automatic closure of interactive sessions in REANA can be customized by configuring
+[Helm values](https://github.com/reanahub/reana/tree/master/helm/reana).
+The following Helm values allow you to control the behavior of the automatic closure feature:
```

Comment:
```suggestion
The automatic closure of inactive interactive sessions in REANA can be customized by means of the following
[Helm values](https://github.com/reanahub/reana/tree/master/helm/reana):
```

To avoid repetition of _automatic closure_